.bit/responses/4.6-Week 4 Step 6.md

```diff
@@ -0,0 +1,22 @@
+---
+files: n/a
+stepType: checks
+scripts: 4.6.spec.js
+week: 4
+step: 6
+name: Week 4 Step 6
+---
+
+Week 4 Step 6 ⬤⬤⬤⬤⬤⬤◯◯◯ | 🕐 Estimated completion: 5-20 minutes
+
+## One Cat isn't enough ~ Meow
+
+Demo: [🐱](https://week4step6.emilychen10.repl.co/)
+
+### ✅  Task:
+- [ ] Create an async function `getCatPic(name)` in `twocatz.js` that takes the input `name` and outputs the base 64 of the cat picture with the `name` on it produced by the [Cataas API](https://cataas.com/)
```

Comment:
Specify that it's an Azure Function, and NOT in the frontend, and to use their previously coded function